Fundamentals/Backoffice/Login/index-v9.md

```diff
@@ -0,0 +1,100 @@
+
+![Login screen](images/backoffice-login.png "The login screen has a greeting, username/password field and optionally a 'Forgotten password' link.")
+
+The **login** screen contains a **Greeting**, **Email**, **Password** field and optionally a **Forgotten password** link
+
+Below, you will find instructions on how to customise the login screen.
+
+## Greeting
+
+The login screen features a greeting, which you can personalize by changing the language file of your choice. For example for en-US you would add the following keys to: `~/umbraco/config/lang/en_us.xml`
+
+```xml
+<area alias="login">
+    <key alias="greeting0">Happy super Sunday</key>
+    <key alias="greeting1">Happy manic Monday </key>
+    <key alias="greeting2">Happy tubular Tuesday</key>
+    <key alias="greeting3">Happy wonderful Wednesday</key>
+    <key alias="greeting4">Happy thunderous Thursday</key>
+    <key alias="greeting5">Happy funky Friday</key>
+    <key alias="greeting6">Happy Caturday</key>
+</area>
+```
+
+You can customize other text in the login screen as well, grab the default values from `~/Umbraco/Config/Lang/en.xml` and copy the keys you want to translate into your `~/Umbraco/Config/Lang/MYLANGUAGE.xml` file.
+
+## Password reset
+
+The **Forgotten password?** link allows your backoffice users to reset their password. To use this feature, you will need to add the following key to the `"security"` section in the `appsettings.json` file:
+
+```json
+"Security": { 
+        "AllowPasswordReset": true
+      },
+```
+
+Set it to `true` to enable the password reset feature, and `false` to disable the feature.
+
+You will also need to configure an SMTP server in your `appsettings.json` file. When you get a successful result on the SMTP configuration when running a health check in the backoffice, you are good to go!
+
+An example:
+
+```json
+"Global": {
+        "Id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+        "Smtp": {
+          "From": "noreply@test.com",
+          "Host": "127.0.0.1",
+          "Username": "username",
+          "Password": "password"
+        }
+      },
+```
+
+## Background image
+
+It is possible to customize the background image for the backoffice login screen by adding the `"Content"` section in the `appsettings.json` file:
+
+```json
+"Content": {
+        "LoginBackgroundImage": "assets/img/login.jpg"
+      }
+```
+
+## `appsettings.json` file
+Your `appsettings.json` file should now look like this:
+```json
+ "Umbraco": {
+    "CMS": {
+      "Hosting": {
+        "Debug": false
+      },
+      "Global": {
+        "Id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+        "Smtp": {
+          "From": "noreply@test.com",
+          "Host": "127.0.0.1",
+          "Username": "username",
+          "Password": "password"
+        }
+      },
+     "Security": { 
+        "AllowPasswordReset": true
+      },
+      "Content": {
+        "LoginBackgroundImage": "assets/img/login.jpg"
+      }
+    }
+ }
+```
```

Comment:
I don't think there is a need to add the full config file in this article.
The configs and settings described in this article, are all something you can do independently from each other.
Showing each individual config setting including keys and groups should be enough.

What do you think?

Reply:
Yeah, Good idea :)